Code/Framework/AzToolsFramework/Tests/Prefab/Spawnable/SpawnableTicketTests.cpp

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/UnitTest/TestTypes.h>
+#include <AzFramework/Spawnable/SpawnableEntitiesInterface.h>
+#include <Prefab/Spawnable/SpawnableTestFixture.h>
+
+namespace UnitTest
+{
+    class SpawnableTicketTests : public SpawnableTestFixture
+    {
+    };
+
+    TEST_F(SpawnableTicketTests, CreateAndDestroyEmptyTicket)
+    {
+        AzFramework::EntitySpawnTicket ticket;
+    }
+
+    TEST_F(SpawnableTicketTests, CreateAndDestroyAssignedTicket)
+    {
+        AzFramework::EntitySpawnTicket ticket(CreateSpawnableAsset(32));
+    }
```

Comment:
Looks like these tests can be deleted since they are pretty much covered in the following 2 tests.

Reply:
Fair argument for reducing the time spend in tests. I added these to stick with the principle of testing exactly one thing per test as much as possible. This means that if the later test fail, these test confirm it's not because of construction/destruction of tickets.